### PR TITLE
Fix SPEC changelog, and check for warnings from rpmbuild - v1.8.x

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: buildlib/azure-pipelines-pr.yml [hunk 1]
```diff
@@ -131,21 +131,28 @@ stages:
...
       - job: build_rpm
-        displayName: build tarball and source rpm
+        displayName: Build tarball and source rpm
         container: fedora
         steps:
           - checkout: self
             clean: true
 
-          - bash: ./autogen.sh
+          - bash: ./autogen.shq
```
Comment (hoopoepg) [inline buildlib/azure-pipelines-pr.yml:140]: shq?

File: buildlib/azure-pipelines-pr.yml [hunk 2]
```diff
@@ -144,8 +144,15 @@ stages:
               set -eE
               gcc --version
               ./contrib/configure-release
-              ./contrib/buildrpm.sh -s -t -b
-            displayName: Build tarball
+              stdbuf -e0 -o0 ./contrib/buildrpm.sh -s -t -b |& tee rpmbuild.log
+              pattern='^warning: '
```
Comment (mike-dubman) [inline buildlib/azure-pipelines-pr.yml:148]: i dont think pattern ```^warning:``` is correct, it searches from col #1 of str, while warning: in log file prefixed with file:line

Comment (yosefe) [reply buildlib/azure-pipelines-pr.yml:148]: the log file does not have timestamp prefix (only AZP adds it later)
i any case, i've tested that when SPEC is wrong, it finds it: https://github.com/openucx/ucx/runs/845557002

**Comment by shamisp** [comment]
👍

**Pull request merged**